Trailing comma for extensions definition of `quarkus-maven-plugin:create` causes NPE

Trailing comma for extensions definition of `quarkus-maven-plugin:create` causes NPE and mvn command fail

```
mvn io.quarkus:quarkus-maven-plugin:999-SNAPSHOT:create \
    -DprojectGroupId=io.quarkus.qe \
    -DprojectArtifactId=scenario-101-getting-started \
    -DprojectVersion=1.0.0-SNAPSHOT \
    -DclassName="io.quarkus.qe.hello.GreetingResource" \
    -Dextensions=io.quarkus:quarkus-smallrye-jwt,
```
```
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------< org.apache.maven:standalone-pom >-------------------
[INFO] Building Maven Stub Project (No POM) 1
[INFO] --------------------------------[ pom ]---------------------------------
[INFO]
[INFO] --- quarkus-maven-plugin:999-SNAPSHOT:create (default-cli) @ standalone-pom ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.214 s
[INFO] Finished at: 2019-06-04T16:04:03+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:999-SNAPSHOT:create 
  (default-cli) on project standalone-pom: Execution default-cli of goal io.quarkus:quarkus-
  maven-plugin:999-SNAPSHOT:create failed.: NullPointerException -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

--- a/devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java
+++ b/devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java
@@ -33,6 +33,7 @@ import java.io.File;
 import java.io.IOException;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Objects;
 import java.util.Set;
 import java.util.stream.Collectors;
 
@@ -141,6 +142,7 @@ public class CreateProjectMojo extends AbstractMojo {
 
         boolean success;
         try {
+            sanitizeExtensions();
             final SourceType sourceType = CreateProject.determineSourceType(extensions);
             sanitizeOptions(sourceType);
 
@@ -290,8 +292,10 @@ public class CreateProjectMojo extends AbstractMojo {
                 path = "/" + path;
             }
         }
+    }
 
-        extensions = extensions.stream().map(String::trim).collect(Collectors.toSet());
+    private void sanitizeExtensions() {
+        extensions = extensions.stream().filter(Objects::nonNull).map(String::trim).collect(Collectors.toSet());
     }
 
     private void printUserInstructions(File root) {

